### PR TITLE
New CSS Questions

### PR DESCRIPTION
**Pull request by nataliepina**
# Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Questions included:
* Can you explain the difference between px, em and rem as it related to font sizing?
* Can you give me an example use case for a pseudo class? 
* Could you provide me with an example of some block level elements and inline elements?

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] New Question
- [ ] Revision of an existing question
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate) 

# Checklist:

- [x] My content follows the style guidelines of this project
- [x] I have performed a self-review of my own content

Pull requests should be thought of as a conversation. There will be some back and forth when trying to get code merged into this or any other project. With all but the simplest changes you can and should expect that the maintainers of the project will request changes to your code. Please be aware of that and check in after you open your PR in order to get your code merged in cleanly. 

Thanks!

### CODE REVIEW

**Comment by Solojoey** [comment]
Not ready for that yet still trying to get a laptop

**Comment by vvscode** [comment]
it's nice to have per question PRs

File: src/questions/css-questions.md [hunk 1]
```diff
@@ -36,3 +36,6 @@ permalink: /questions/css-questions/index.html
 * Have you ever worked with retina graphics? If so, when and what techniques did you use?
 * Is there any reason you'd want to use `translate()` instead of *absolute positioning*, or vice-versa? And why?
 * How is clearfix css property useful?
+* Can you explain the difference between px, em and rem as it related to font sizing?
```
Comment (roblarsen) [inline src/questions/css-questions.md:39]:
```suggestion
* Can you explain the difference between px, em and rem as they relate to font sizing?
```

File: src/questions/css-questions.md [hunk 2]
```diff
@@ -36,3 +36,6 @@ permalink: /questions/css-questions/index.html
...
+* Can you give me an example use case for a pseudo class?
```
Comment (roblarsen) [inline src/questions/css-questions.md:40]:
```suggestion
* Can you give an example of a pseudo class? Can you provide an example use case for a pseudo class? 
```

File: src/questions/css-questions.md [hunk 3]
```diff
@@ -36,3 +36,6 @@ permalink: /questions/css-questions/index.html
 * Have you ever worked with retina graphics? If so, when and what techniques did you use?
 * Is there any reason you'd want to use `translate()` instead of *absolute positioning*, or vice-versa? And why?
 * How is clearfix css property useful?
+* Can you explain the difference between px, em and rem as it related to font sizing?
+* Can you give me an example use case for a pseudo class? 
+* Could you provide me with an example of some block level elements and inline elements?
```
Comment (roblarsen) [inline src/questions/css-questions.md:41]:
```suggestion
* What is the difference between a block level element and an inline element. Can you provide examples of each type of element?
```

**Comment by roblarsen** [comment]
@nataliepina Good questions. I made some editorial suggestions.

**Comment by nataliepina** [comment]
Changes have been made, thank you for the comments!

**Comment by roblarsen** [comment]
@nataliepina thanks!

**Pull request merged**